pkg/controller/ibmblockcsi/syncer/csi_node.go

```diff
@@ -97,6 +97,7 @@ func (s *csiNodeSyncer) ensurePodSpec() corev1.PodSpec {
 		Containers:         s.ensureContainersSpec(),
 		Volumes:            s.ensureVolumes(),
 		HostIPC:            true,
+		HostNetwork: 		true,
```

Comment:
Is it mandatory? Please also consider the impact.

Reply:
Doesn't work without it:
```bash-4.2# iscsiadm -m discoverydb -t st -p 172.18.2.13:3260 --discover
iscsiadm: can not connect to iSCSI daemon (111)!
iscsiadm: Cannot perform discovery. Initiatorname required.
iscsiadm: Could not perform SendTargets discovery: could not connect to iscsid```